packages/entrykit/src/onboarding/ConnectedSteps.tsx

```diff
@@ -129,7 +129,7 @@ export function ConnectedSteps({ userClient, initialUserAddress }: Props) {
     <div
       className={twMerge(
         "px-8 flex flex-col",
-        focusedId && "divide-y divide-neutral-800",
+        !focusedId && "divide-y divide-neutral-800",
```

Comment:
intentional?

Reply:
There is now a generalized logic for setting a focused "screen", like in the case of "Deposit" form. So we want to show the divider when there is no screen focused i.e. we are seeing the initial "steps" page. But if we go inside a "Deposit" form page, we don't want to have the divider while keeping the rest of the styling the same.

Reply:
right, actually did this another way, and put the whole "Deposit" form under one container. It was trying to divide the top "Go back / Close button" part from the actual form part. So removed the conditional styling check.